knowledge-base/redhat-actions/oc-login/action-security.yml

```diff
@@ -0,0 +1,5 @@
+name: 'OpenShift Login'
+outbound-endpoints:
+  - fqdn: openshift-online.com
```

Comment:
https://stackoverflow.com/questions/46417232/what-server-url-should-be-used-for-the-oc-login-command-when-using-openshifts